types/api-error-handler/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://github.com/expressjs/api-error-handler
 // Definitions by: Tanguy Krotoff <https://github.com/tkrotoff>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.3
```

Comment:
I had to bump the TS version of Express and all its dependants because I needed to use default generics in the Express types.